### ISSUE TITLE
Let users deactivate/delete their accounts

### ISSUE THREAD

**Issue by BMariscal** [FightPandemics/FightPandemics#1148]
Task should process account deletion requests where all posts and account-related information is deleted.

**Comment by ddveloper** [comment]
@Zaydme a big chance we will need this to enable Facebook API more scope access, check #2069 for details

**Comment by Zaydme** [comment]
@ddveloper I have already a PR open but it is very old/outdated and horribly coded 😅
But I can revisit it if it was needed, and legally talking I think it is.
Let me know if you want me to revisit/finish that PR

**Comment by ddveloper** [comment]
Yes, @Zaydme I think we need to prioritize 🌶️ this issue now for expanding our Facebook scopes
